### PR TITLE
Internationalize chat message

### PR DESCRIPTION
**Pull request by NonSwag**
## Description

- introduced [i18n lib](https://github.com/TheNextLvl-net/core/tree/master/i18n)
- added german and english translations
- translated all chat messages

image
image

## Checklist

- [x] I included all information required in the sections above
- [x] I tested my changes and approved their functionality
- [x] I ensured my changes do not break other parts of the code

### CODE REVIEW

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -15,6 +16,7 @@ bstats = { group = "org.bstats", name = "bstats-bukkit", version.ref = "bstats"
 cloud-annotations = { group = "org.incendo", name = "cloud-annotations", version.ref = "cloud-annotations" }
 cloud-minecraft-extras = { group = "org.incendo", name = "cloud-minecraft-extras", version.ref = "cloud-minecraft" }
 cloud-paper = { group = "org.incendo", name = "cloud-paper", version.ref = "cloud-minecraft" }
+core-internationalization = { group = "net.thenextlvl.core", name = "i18n", version.ref = "core-i18n" }
```
Comment (TheMeinerLP) [inline gradle/libs.versions.toml:19]: Please use pure Adventure, for i18n we will not rely on unpopular 3Party lib

**Review by TheMeinerLP — CHANGES_REQUESTED** [review]
We prefer a way like this: https://github.com/PaperMC/Velocity/pull/1108/files

**Comment by NonSwag** [comment]
> We prefer a way like this: https://github.com/PaperMC/Velocity/pull/1108/files

velocity does not even support minimessage in configurations

**Comment by TheMeinerLP** [comment]
The linked PR is that pr solve that problem

**Pull request closed without merging**